lib/client/commands/login.go

```diff
@@ -84,16 +85,23 @@ func (l *Login) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	// Right now recreating the logger because ssh requires a klog.Logger and it only gets called once and immediately is
+	// coerced into an interface type, not sure if I should just typecast here or deal with propagation
```

Comment:
Basically, libraries are written so they work with any logging library that implements the `logger.Logger` interface.
Just change the type of the struct to be `logger.Logger`, pass in the logger defined by base.

(otherwise, the flags passed on the command line to control logging won't work - if the libraries you just recently merged in take a concrete klog, they should be changed to take an abstract logger instead - may have missed it during review)